examples/30_extended/fetch_runtimes.py

```diff
@@ -66,11 +70,11 @@ def print_compare_runtimes(measures):
 ############################################################################
 # Case 1: Running a Random Forest model on an OpenML task
 # *******************************************************
-# We'll run a Random Forest model and obtain an OpenML run-object. We can
+# We'll run a Random Forest model and obtain an OpenML run object. We can
 # see the evaluations recorded per fold for the dataset and the information
 # available for this run.
 
-run1 = openml.runs.run_model_on_task(model=clf, task=task)
+run1 = openml.runs.run_model_on_task(model=clf, task=task, dataset_format="array")
```

Comment:
I left it like this for a couple of reasons.

Firstly, based on local runs, for certain datasets, the difference in runtimes noticeable `visually` was more pronounced with arrays instead of data frames. Though of course, this is an entirely empirical observation conditioned on my local machine.

Secondly, while testing for various datasets/tasks, the `array` formatting allowed for quick experimentation without complicating the sklearn model/pipeline.